tools/attest/attest.go

```diff
@@ -130,8 +130,9 @@ func parseBytes(name string, in io.Reader, inform string, byteSize int) ([]byte,
 	}
 }
 func main() {
-	logger.Init("", *verbose, false, os.Stderr)
```

Comment:
Incorrect location to init logger since verbose flag is never parsed so this was always ignored.